app/scheduler/onlinetesting/EvaluateOnlineTestResultJob.scala

```diff
@@ -39,7 +40,14 @@ object EvaluatePhase2ResultJob extends EvaluateOnlineTestResultJob[Phase2PassMar
   val phase = Phase.PHASE2
 }
 
-trait EvaluateOnlineTestResultJob[T <: PassMarkSettings] extends SingleInstanceScheduledJob {
+object EvaluatePhase3ResultJob extends EvaluateOnlineTestResultJob[Phase3PassMarkSettings] with EvaluatePhase3ResultJobConfig {
+  val evaluateService = EvaluatePhase3ResultService
+  val phase = Phase.PHASE3
+  override val errorLog = (app: ApplicationReadyForEvaluation) =>
+    s"${app.applicationId}, Launchpad interview Id: ${app.activeLaunchpadTest.map(_.interviewId)}"
```

Comment:
The interview ID is not particularly useful here, because almost everyone's invited to the same interview, if you want something that identifies a specific set of tests i'd suggest using the token (AKA launchpad invite ID)